_targets.R

```diff
@@ -6,54 +6,49 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 #site numbers
-sites = c("01427207", "01432160","01435000", "01436690", "01466500")
+#sites = c("01427207", "01432160","01435000", "01436690", "01466500")
 #parameter codes for query
-parameterCd = '00010' 
-startDate="2014-05-01" 
-endDate="2015-05-01"
-#parameters for plotting
-p_width <- 12
-p_height <- 7
-p_units <- "in"
+
+
 
 p1_targets_list <- list(
+  # tar_target(
+  #   create_directories,
+  #   create_dirs(site_num = c("01427207", "01432160","01435000", "01436690", "01466500")),
+  #   format = "file"
+  # ),
   tar_target(
-    create_directories,
-    create_dirs(sites),
-    format = "file"
-  ),
-  tar_target(
-    site_data_1,
-    download_nwis_site_data(sites[1], create_directories,parameterCd, startDate, endDate),
+    nwis_01427207_data_csv,
+    download_nwis_site_data("01427207", create_directories),
     format = "file"
   ),
```

Comment:
I think here you will want to drop the create_directories input as you don't have that target being built anymore. The standard approach for USGS pipelines is to have the output file paths set in the targets so you can see right from the target file where it is being saved.  So here you could consider replacing create_directories with the target output directory.

Reply:
Whoops, I was doing this late last night and I forgot to remove the ```create_directories``` input, I don't think it's doing anything. Also, I have added the output file path as an input to each site target.